redfish-client/etc/redfish-client.conf

```diff
@@ -0,0 +1,2 @@
+[redfish-client]
+templates_path = /usr/share/redfish-client/templates
```

Comment:
I was more thinking of a pb Macro such as a PBTEMPLATEPATH and define this one depending on the distros in the pbfilter dirs. We need to talk about that.